common/src/main/java/com/wynntils/core/features/overlays/overlaySizes/FixedOverlaySize.java

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
I'm really allergic to upper case letters in package names. 🤧  I don't think I've ever seen that before. In this case I think it's easy to resolve by just renaming the package `sizes`, since it is below `overlays` anyway.